.config/tsaoptions.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
What's this file?

Reply:
It is used by the sdl template. More info here:
https://www.osgwiki.com/wiki/Package_ES_SDL_Templates#TSA_config

Reply:
Specifically, it controls the bugs that get filed when an issue is found. For example, if PoliCheck hits an issue, the Pipeline will file a bug in our path for us to triage.